docs/_quartodoc.yml

```diff
@@ -178,9 +179,12 @@ quartodoc:
           contents:
             - render.renderer.Renderer
             - render.renderer.RendererBase
-            - render.renderer.Jsonifiable
-            - render.renderer.ValueFn
-            - render.renderer.AsyncValueFn
+            - name: render.renderer.Jsonifiable
+              dynamic: false
```

Comment:
Why is it important that these are `dynamic: false`? For better performance, or correctness?

Reply:
In this case and the other one in this section, `dynamic: true` leads to cyclic reference errors in griffe